src/fides/api/models/privacy_experience.py

```diff
@@ -314,27 +284,6 @@ def get_related_privacy_notices(
 
         return notices
 
-    @classmethod
-    def create(
-        cls: Type[PrivacyExperience],
-        db: Session,
-        *,
-        data: dict[str, Any],
-        check_name: bool = False,
-    ) -> PrivacyExperience:
-        """Create a privacy experience and the clone this record into the history table for record keeping"""
-        privacy_experience = super().create(db=db, data=data, check_name=check_name)
-
-        # create the history after the initial object creation succeeds, to avoid
-        # writing history if the creation fails and so that we can get the generated ID
-        data.pop("id", None)
-        history_data = {
-            **data,
-            "privacy_experience_id": privacy_experience.id,
-        }
-        PrivacyExperienceHistory.create(db, data=history_data, check_name=False)
-        return privacy_experience
-
```

Comment:
I don't think we need any special logic here for create any longer, same for update below.